cyclonedx/cyclonedx.go

```diff
@@ -98,6 +100,11 @@ func processPurlsIntoSBOMSchema1_1(results []types.Coordinate) string {
 			return ""
 		}
 
+		// IQ requires a v before versions, so add one if it doesn't exist
+		if !strings.HasPrefix(purl.Version, "v") {
```

Comment:
I was scratching my head a while until I realized the cyclonedx code is ONLY called for IQ. Not sure I see a good way to make that clearer, via packaging or some other way - thought about if `cyclonedx*.go` should move under the `iq` package, but not sure....more asking the question.

Reply:
It likely makes sense to just create a new project like @fitzoh or @zendern suggested: `go-sona-types`, and have sort of "common" functionality in that one.